presto-main/src/main/java/com/facebook/presto/server/security/LdapFilter.java

```diff
@@ -144,9 +145,7 @@ public Principal getUserPrincipal()
                 return;
             }
             catch (AuthenticationException e) {
-                // Log exception since the error message from LDAP server is not very friendly to be returned to the user
-                LOG.debug("LDAP Authentication failed with exception: %s", e);
-                throw new RuntimeException("Authentication failed. Invalid credentials.");
+                throw new RuntimeException("Invalid credentials. Detailed Message: " + e.getMessage());
```

Comment:
Perhaps we should get rid of the "Message" word too and have error messages in the following format: `Invalid credentials: " + e.getMessage()`.